persistence/trees/trees_test.go

```diff
@@ -0,0 +1,18 @@
+package trees
```

Comment:
The list of headers doesn't seem to be exhaustive and I think it's a non-trivial task to figure out what to test. While you're working on this, you probably have the most context of what we need, which will be lost later.

Can you please do one of:
1. Add more test function headers on what we think we need
2. Create a new github issue (or see if one exists) and link to it here. Potentially add a deliverable to leverage ChatGPT to see what else we should test for

The lack of testing in this is one of the few things that keep me up at night so appreciate you understanding :)

Reply:
#836 covers tests and mentions this refactor in the description. Is that sufficient here instead?

Reply:
Yea, just link to the ticket in the `trees_test` file